backend/api/tokenVerificationApi.js

```diff
@@ -23,7 +23,7 @@ const tokenVerificationApiFactory = client => {
         log.debug(`Token verification result is ${result}`)
         return result
       })
-      .catch(processError)
+      .catch(() => false)
```

Comment:
this was causing the promise not to resolve properly.  Fine not to throw error as has been logged